internal/daemon/hostsidemanager.go

```diff
@@ -296,6 +356,24 @@ func (d *HostSideManager) Serve(ctx context.Context, listener net.Listener) erro
 		wg.Done()
 	}()
 
+	wg.Add(1)
+	go func() {
+		d.log.Info("Starting ping client")
+		ticker := time.NewTicker(1 * time.Second)
+		defer ticker.Stop()
+		defer wg.Done()
+
+		for {
+			select {
```

Comment:
Can we have a pattern similar to the above like starting manager and starting device plugin server?

Reply:
done